codex/validation.nim

```diff
@@ -107,7 +118,11 @@ proc markProofsAsMissing(validation: Validation) {.async.} =
     await validation.markProofAsMissing(slotId, previousPeriod)
 
 proc run(validation: Validation) {.async.} =
-  trace "Validation started"
+  logScope:
+    groups = validation.config.groups
+    groupIndex = validation.config.groupIndex
+  trace "Validation started", currentTime = validation.clock.now,
+    currentTime = validation.clock.now.fromUnix
```

Comment:
Perhaps `currentTime` can be dropped as we can use the timestamp of the log itself. It was just convenient when using `-d:chronicles_sinks=textlines[notimestamps,stdout]` as for readability reasons, I sometimes prefer chronicle logs without timestamps. But in principle we can remove `currentTime`. For `groups` and `groupsIndex`, I wanted to annotate each single validator log entry so that I can easily identify which validator it is. Similar to  https://github.com/codex-storage/nim-codex/pull/922#discussion_r1869687819.